content/docs/2.0/scalers/azure-event-hub.md

```diff
@@ -34,7 +34,16 @@ triggers:
 
 ### Authentication Parameters
 
-Not supported yet.
+You can authenticate by using connection string authentication.
+
+**Connection String Authentication:**
+
+- `connection` - Connection string for the Azure Service Bus Namespace.
```

Comment:
```suggestion
- `connection` - Connection string for the Azure Event Hub.
```

Reply:
This applies for all versions 2.0.x => 2.8.x.

Reply:
Even copy pasting is something I'm not good at